enabled: True
python: |
  def break6():
    res = False
    while True:
      try:
        x = int(1)
        print(x)
      except ValueError:
        print("Oops!  That was not a valid number.  Try again...")
      else:
        res = True
        break
    
    return res